python/python/glide/async_commands/server_modules/json.py

```diff
@@ -320,6 +320,61 @@ async def forget(
     )
 
 
+async def resp(
+    client: TGlideClient, key: TEncodable, path: Optional[TEncodable] = None
+) -> TJsonUniversalResponse[
+    Optional[Union[bytes, int, List[Optional[Union[bytes, int]]]]]
+]:
+    """
+    Retrieve the JSON value at the specified `path` within the JSON document stored at `key`.
+    The returning result is in the Valkey or Redis OSS Serialization Protocol (RESP).\n
+    JSON null is mapped to the RESP Null Bulk String.\n
+    JSON Booleans are mapped to RESP Simple string.\n
+    JSON integers are mapped to RESP Integers.\n
+    JSON doubles are mapped to RESP Bulk Strings.\n
+    JSON strings are mapped to RESP Bulk Strings.\n
+    JSON arrays are represented as RESP arrays, where the first element is the simple string [, followed by the array's elements.\n
+    JSON objects are represented as RESP object, where the first element is the simple string {, followed by key-value pairs, each of which is a RESP bulk string.\n
```

Comment:
Remove `\n`s
A user of GLIDE py client doesn't see, doesn't know and shouldn't know RESP types and such things. Simplify this and skip obvious things, like number represented by a number.
Delete lines 330 to 337 and leave a comment that a map represented by a 2D array of K-V pairs.

Reply:
I am using \n because it looks better in the ide

Reply:
User of glide is user of redis / valkey 
They should know what resp is, since it's pretty basic in the server side + mentioned multiple times in our commands documentation

Reply:
I want to live it as informative as I can, imo it simplifies and explains really good all cases 🙁

Reply:
We don't do this in other commands, why start here? Your doc describes how json types mapped to RESP types, nothing said about how it is mapped to python types. Such doc is useless for an arbitrary GLIDE py user.
Re: `\n` - we need to follow a python docstring format guidelines. It is sad that py client has neither doc linter nor doc generator/renderer.

Reply:
BTW
> JSON Booleans are mapped to RESP Simple string

Is that true? Please, test. If yes, we need to do a value conversion or mention in our docs.

Reply:
But I did test it

Reply:
And I disagree, in each command we are very detailed about what is being returned, here it's a bit different bc we explain EACH type and what's it's mapped to